build.sh

```diff
@@ -22,8 +22,7 @@ POMS=(
     "../repositories/SS_ProgramManagementClient/pom.xml"
     "../repositories/SS_SharedSecurity/pom.xml"
     "../repositories/TDS_Student/ResourceBundler/pom.xml"
-    "../repositories/TDS_ItemRenderer/shared-blackbox/pom.xml"
-    "../repositories/TDS_ItemRenderer/TDS.ItemRenderer/pom.xml"
+    "../repositories/TDS_ItemRenderer/pom.xml"
```

Comment:
Now that TDS_ItemRenderer has a parent POM, we need to build from the parent rather than building the submodules separately.